apiserver/params/network.go

```diff
@@ -152,10 +152,9 @@ type NetworkConfig struct {
 	// auto-start.
 	NoAutoStart bool `json:"no-auto-start,omitempty"`
 
-	// ConfigType, if set, defines what type of configuration to use.
-	// See network.AddressConfigType for more info. If not set, for
-	// backwards-compatibility, "dhcp" is assumed.
-	ConfigType string `json:"config-type,omitempty"`
+	// ConfigMethod defines what type of configuration to use.
+	// See network.AddressConfigMethod for more info.
+	ConfigMethod string `json:"config-type,omitempty"`
```

Comment:
Note that the name is updated for congruence, but the tag preserves backward-compatibility.